src/components/Search/SearchView.tsx

```diff
@@ -15,8 +18,16 @@ import SearchViewInit from './SearchViewInit';
 function SearchView(props: SearchViewProps) {
   const { focused, handleParentClick, handleFocus } = useFocusInput();
 
+  const navigate = useNavigate();
+
+  const goToResultsPage = () => {
+    navigate('/results', { replace: false });
+  };
+
   const { searchResults, selectedRevisions } = props;
 
+  const compare = 'compare';
```

Comment:
I'm just wondering why you added this variable?

Reply:
messaged you.